tab.js

```diff
@@ -163,90 +162,85 @@
      * @param {event} event 
      */
     function keyHandler(event) {
-        if ((event.keyCode || event.which) === 27) {
+        if ((event.keyCode || event.which) === VK_ESC) {
             toggleTabList();
         }
     }
-    
+
     const chromeTabModule = {
-        
+
         allOpenedTabs: [],
-        
+
         /**
          * navigates the tab to corresponding user selection
          */
         switchTab: (event) => {
             let selectedTab = {};
-            
+
             Array.from(event.currentTarget.attributes)
-            .forEach(data => {
-                if (data.name !== 'class') {
-                    selectedTab['tabId']
-                    ? selectedTab['windowId'] = data.value
-                    : selectedTab['tabId'] = data.value;
-                }
-            });
-            
-            chrome.extension.sendMessage({type: 'switchTab', selectedTab}, function() {});
-            toggleTabList();
+                .forEach(data => {
+                    if (data.name !== 'class') {
+                        selectedTab['tabId'] ?
```

Comment:
this part also thought of refactoring. Check whether is it make sense bro.

`selectedTab[selectedTab.tabId ? 'windowId' : 'tabId'] = data.value;`

Reply:
makes sense.. power of JS 💪